### PR TITLE
fix(select): cleanup vars

### LINKED ISSUE #2133: Remove select toggle-button custom disabled css
After https://github.com/patternfly/patternfly-next/issues/2127 is merged, we can remove the custom CSS in select that sets a transparent background when that element is set to `disabled` since those styles will come from the plain button css.

### PR DESCRIPTION
**Pull request by christiemolloy**
closes #2133 and #2618 

## Breaking Changes

Will fill in this section once the PR is reviewed.

### CODE REVIEW

File: src/patternfly/components/Select/select.scss [hunk 1]
```diff
@@ -11,25 +11,19 @@
...
   --pf-c-select__toggle--Color: var(--pf-global--Color--100);
-  --pf-c-select__toggle--hover--BorderBottomColor: var(--pf-global--active-color--100);
-  --pf-c-select__toggle--active--BorderBottomWidth: var(--pf-global--BorderWidth--md);
-  --pf-c-select__toggle--active--BorderBottomColor: var(--pf-global--active-color--100);
-  --pf-c-select__toggle--expanded--BorderBottomWidth: var(--pf-global--BorderWidth--md);
-  --pf-c-select__toggle--expanded--BorderBottomColor: var(--pf-global--active-color--100);
+  --pf-c-select__toggle--hover--before--BorderBottomColor: var(--pf-global--active-color--100);
+  --pf-c-select__toggle--active--before--BorderBottomWidth: var(--pf-global--BorderWidth--md);
+  --pf-c-select__toggle--active--before--BorderBottomColor: var(--pf-global--active-color--100);
+  --pf-c-select__toggle--expanded--before--BorderBottomWidth: var(--pf-global--BorderWidth--md);
```
Comment (mcoker) [inline src/patternfly/components/Select/select.scss:23]:
```suggestion
  --pf-c-select__toggle--m-expanded--before--BorderBottomWidth: var(--pf-global--BorderWidth--md);
```

File: src/patternfly/components/Select/select.scss [hunk 2]
```diff
@@ -11,25 +11,19 @@
...
+  --pf-c-select__toggle--expanded--before--BorderBottomColor: var(--pf-global--active-color--100);
```
Comment (mcoker) [inline src/patternfly/components/Select/select.scss:24]:
```suggestion
  --pf-c-select__toggle--m-expanded--before--BorderBottomColor: var(--pf-global--active-color--100);
```

File: src/patternfly/components/Select/select.scss [hunk 3]
```diff
@@ -188,48 +183,36 @@
...
+    border: var(--pf-c-select__toggle--before--BorderWidth) solid;
+    border-color: var(--pf-c-select__toggle--before--BorderTopColor) var(--pf-c-select__toggle--before--BorderRightColor) var(--pf-c-select__toggle--before--BorderBottomColor) var(--pf-c-select__toggle--before--BorderLeftColor);
   }
 
   &:hover {
     &::before {
-      --pf-c-select__toggle--BorderBottomColor: var(--pf-c-select__toggle--hover--BorderBottomColor);
+      --pf-c-select__toggle--before--BorderBottomColor: var(--pf-c-select__toggle--hover--before--BorderBottomColor);
     }
   }
```
Comment (mcoker) [inline src/patternfly/components/Select/select.scss:195]: Looks like we're missing a `:focus` state - would you mind adding one in this PR?

File: src/patternfly/components/Select/select.scss [hunk 4]
```diff
@@ -188,48 +183,36 @@
...
 
-      border-bottom-width: var(--pf-c-select__toggle--active--BorderBottomWidth);
+      border-bottom-width: var(--pf-c-select__toggle--active--before--BorderBottomWidth);
     }
   }
 
   .pf-m-expanded > & {
     &::before {
-      --pf-c-select__toggle--BorderBottomColor: var(--pf-c-select__toggle--expanded--BorderBottomColor);
+      --pf-c-select__toggle--before--BorderBottomColor: var(--pf-c-select__toggle--expanded--before--BorderBottomColor);
```
Comment (mcoker) [inline src/patternfly/components/Select/select.scss:207]:
```suggestion
      --pf-c-select__toggle--before--BorderBottomColor: var(--pf-c-select__toggle--m-expanded--before--BorderBottomColor);
```

File: src/patternfly/components/Select/select.scss [hunk 5]
```diff
@@ -188,48 +183,36 @@
...
 
-      border-bottom-width: var(--pf-c-select__toggle--expanded--BorderBottomWidth);
+      border-bottom-width: var(--pf-c-select__toggle--expanded--before--BorderBottomWidth);
```
Comment (mcoker) [inline src/patternfly/components/Select/select.scss:209]:
```suggestion
      border-bottom-width: var(--pf-c-select__toggle--m-expanded--before--BorderBottomWidth);
```

File: src/patternfly/components/Select/select.scss [hunk 6]
```diff
@@ -395,18 +359,17 @@
   color: var(--pf-c-select__menu-item--Color);
   text-align: left;
   white-space: nowrap;
-  background-color: transparent; // this can be removed if the shield in _base.css works for FF
```
Comment (mcoker) [inline src/patternfly/components/Select/select.scss:398]: This is needed now that we removed `* {background-color: transparent;}`. Let's chat about it tomorrow.

**Comment by mcoker** [comment]
Looks like you have a conflict @christiemolloy

**Comment by christiemolloy** [comment]
updated @mcoker

**Comment by christiemolloy** [comment]
updated @mattnolting i think what you meant was to just swap around where the vars are positioned in the file right ?

**Pull request merged**